apps/condo/domains/ticket/access/TicketFile.js

```diff
@@ -0,0 +1,54 @@
+
+/**
+ * Generated by `createschema ticket.TicketFile 'organization:Text;file?:File;ticket?:Relationship:Ticket:SET_NULL;'`
+ */
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+ 
+
+async function canReadTicketFiles ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) {
+        return {}
+    }
+    return { organization: { employees_some: { user: { id: user.id }, isBlocked: false } } }
+}
+
+async function canManageTicketFiles ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return false
+    if (user.isAdmin) return true
+    if (operation === 'create') {
+        const organizationIdFromTicketFile = get(originalInput, ['organization', 'connect', 'id'])
+        if (!organizationIdFromTicketFile) {
+            return false
+        }
+        const canManageTicketFiles = await checkOrganizationPermission(user.id, organizationIdFromTicketFile, 'canManageTickets')
+        return canManageTicketFiles
```

Comment:
Why just not `return await checkOrganizationPermission…`?
The variable is named as `canManageTicketFiles`, but permission name to be examined is `canManageTickets`.

Reply:
ESlint has a special rule for this: no-return-await. If you are interested in details you can read some [checks](https://kanby.medium.com/%D0%BF%D0%BE%D1%87%D0%B5%D0%BC%D1%83-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D1%8C-return-await-%D0%BF%D0%BB%D0%BE%D1%85%D0%B0%D1%8F-%D0%B8%D0%B4%D0%B5%D1%8F-e87b70015f0c)